pkg/controller/cloneset/sync/cloneset_scale_test.go

```diff
@@ -84,7 +84,7 @@ func TestCreatePods(t *testing.T) {
 					apps.ControllerRevisionHashLabelKey:  "revision_abc",
 					apps.DefaultDeploymentUniqueLabelKey: "revision_abc",
 					"foo":                                "bar",
-					appspub.LifecycleStateKey:            string(appspub.LifecycleStateNormal),
```

Comment:
plz add tests for the pod under preNormal state, possible with Normal, deletion, inplace-update operation cases